src/query/expression/Cargo.toml

```diff
@@ -38,6 +39,7 @@ geo = { workspace = true }
 geos = { workspace = true }
 geozero = { workspace = true }
 hex = "0.4.3"
+hyper = "0.14"
```

Comment:
Hi, it's better not to depend on hyper "0.14" anymore.

I'm currently working on a upgrade for this.

Reply:
It seems arrow-flight depends on tonic 0.11, and tonic 0.11 depends on hyper 0.14. We cannot remove it easily.

Reply:
Maybe we could try upgrade arrow-flight later?

Reply:
Yep, I'm working on this now.